src/VendorViews/ViewVendor.js

```diff
@@ -94,9 +94,10 @@ class ViewVendor extends Component {
   }
 
   update(data) {
+    console.log(data);
```

Comment:
the same question. Maybe it makes sense to use `stripes.logger` in case you need to log something?